src/Tizen.Multimedia.Camera/Camera/CameraSettings.cs

```diff
@@ -1319,40 +1354,35 @@ public ExtraPreviewStreamInfo(int streamId, CameraPixelFormat format, Size size,
         /// Gets the stream Id.
         /// </summary>
         /// <value>The stream Id.</value>
-        /// <since_tizen> 9 </since_tizen>
-        [EditorBrowsable(EditorBrowsableState.Never)]
-        public int StreamId { get; set;}
+        /// <since_tizen> 10 </since_tizen>
+        public int StreamId { get; }
 
         /// <summary>
         /// Gets the extra preview format.
         /// </summary>
         /// <value></value>
```

Comment:
empty `<value>`, either fill it in or remove it, same for the next 2 properties

Reply:
Added.